aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -94,6 +94,64 @@ const connection = new signalR.HubConnectionBuilder()
 > [!NOTE]
 > At this time, there are no configuration options for the MessagePack protocol on the JavaScript client.
 
+## MessagePack Quirks
```

Comment:
Not sure "quirks" is the right name here... very open to suggestions. I hestitate to use "Known Issues" because that (to me) implies that we're going to work on fixing them, which we aren't. I started with "Differences between MessagePack and JSON serialization" but some of these aren't exactly "differences" between JSON and MessagePack, they're just notable things for MessagePack (AOT support, for example didn't feel like a "difference"... though it kind of is... 🤷‍♂️).

Reply:
I think "quirks" describes it well.

Reply:
Use sentence casing for headings. For example, "MessagePack quirks".

Reply:
Cool, wasn't sure if there's a translation concern around using a word like "quirks". I'll update the casing.